app/views/pages/dashboard.html.erb

```diff
@@ -1,14 +1,24 @@
-<h1>Dashboard</h1>
-<p> Your email: <%= current_user.email %> </p>
```

Comment:
Are we sure we don't want something to identify the user visible on the dashboard?